src/validation-dispatcher/single-field-dispatcher.ts

```diff
@@ -0,0 +1,150 @@
+import {
+  FieldValidation,
+  ValidationResult,
+  createDefaultValidationResult,
+} from '../model';
+
+import {
+  arrayContainsEntries,
+  isFunction,
+  isLastIndexInArray,
+  isNorUndefinedOrNull,
+} from '../helper';
+
+interface ValidationParams {
+  values: any;
+  value: any;
+  fieldValidationCollection: FieldValidation[];
+}
+
+const createValidationParams = (
+  values: any,
+  value: any,
+  fieldValidationCollection: FieldValidation[]
+): ValidationParams => ({
+  values,
+  value,
+  fieldValidationCollection,
+});
+
+enum ValidationCheck {
+  ERROR_NOT_EXPECTED,
+  FAILED,
+  SUCCEEDED,
+}
+
+const checkFieldValidationResult = (
+  fieldValidationResult: ValidationResult
```

Comment:
Rename to `validationResult` as interface

Reply:
renamed